ci/k8s/deployment.yml

```diff
@@ -79,12 +79,16 @@ spec:
             port: 8000
           initialDelaySeconds: 30
           timeoutSeconds: 1
+          failureThreshold: 12
```

Comment:
Maybe too high? It'll take 2 mins for an unhealthy pod to be restarted, and there is only one. Default is 3, I would set something like 6.

Reply:
The last liveness issue was due to some requests taking more than 60 seconds, so we want to give at least more than that amount of time. 

Given that up until now the issue has been due to being too stringent, I prefer the 2 mins and see how it goes.

Reply:
Seems reasonable to me.